apps/avatax/e2e/tests/product_without_tax_class.spec.ts

```diff
@@ -0,0 +1,245 @@
+                net {
+                  ...Money
+                }
+              }
+              totalPrice {
+                net {
+                  ...Money
+                }
+                gross {
+                  ...Money
+                }
+                tax {
+                  ...Money
+                }
+              }
+            }
+          }
+        }
+
+        fragment CheckoutErrorFragment on CheckoutError {
+          message
+          field
+          code
+        }
+
+        fragment Money on Money {
+          currency
+          amount
+        }
+      `)
+      .withGraphQLVariables({
+        "@DATA:TEMPLATE@": "UpdateDeliveryMethod:USA",
+      })
+      .expectJson("data.checkoutDeliveryMethodUpdate.checkout.totalPrice.net", {
+        currency: "USD",
+        amount: TOTAL_NET_PRICE_AFTER_SHIPPING,
+      })
+      .expectJson("data.checkoutDeliveryMethodUpdate.checkout.totalPrice.gross", {
+        currency: "USD",
+        amount: TOTAL_GROSS_PRICE_AFTER_SHIPPING,
+      })
+      .expectJson("data.checkoutDeliveryMethodUpdate.checkout.totalPrice.tax", {
+        currency: "USD",
+        amount: TOTAL_TAX_PRICE_AFTER_SHIPPING,
+      })
+      .expectJson("data.checkoutDeliveryMethodUpdate.checkout.shippingPrice.net", {
+        currency: "USD",
+        amount: SHIPPING_NET_PRICE,
+      })
+      .expectJson("data.checkoutDeliveryMethodUpdate.checkout.shippingPrice.gross", {
+        currency: "USD",
+        amount: SHIPPING_GROSS_PRICE,
+      })
+      .expectJson("data.checkoutDeliveryMethodUpdate.checkout.shippingPrice.tax", {
+        currency: "USD",
+        amount: SHIPPING_TAX_PRICE,
+      })
+      .retry();
+  });
+
+  it("checkout should be able to complete", async () => {
+    await testCase
+      .step("Complete checkout")
+      .spec()
+      .post("/graphql/")
+      .withGraphQLQuery(gql`
+        mutation CompleteCheckout($checkoutId: ID!) {
+          checkoutComplete(id: $checkoutId) {
+            order {
+              id
+            }
+            confirmationNeeded
+            confirmationData
+            errors {
+              field
+              message
+              code
+              variants
+              lines
+              addressType
+            }
+          }
+        }
+      `)
+      .withGraphQLVariables({ checkoutId: "$S{CheckoutId}" })
+      .expectJsonMatch({
+        data: {
+          checkoutComplete: {
+            order: {
+              id: string(),
+            },
+          },
+        },
+      })
+      .stores("OrderId", "data.checkoutComplete.order.id")
+      .clean()
+      .post("/grpahql/")
+      .withGraphQLQuery(gql`
+        mutation OrderCancel($orderId: ID!) {
+          orderCancel(id: $orderId) {
+            errors {
+              field
+              message
+              code
+            }
+          }
+        }
+      `)
+      .withGraphQLVariables({
+        orderId: "$S{OrderId}",
+      });
```

Comment:
Why call orderCancel here?

Reply:
I wanted to leave less mess in the environment after running the test, but I forgot it doesn't actually delete the order, just marks the fulfillment as cancelled 😅

Reply:
I removed this

Reply:
In the future, we will add CI CD and restore snapshots so the env will be clean before running tests. 
Also, it's better to leave this order after running the test since it may be helpful for debugging.